requirements/pytorch/adjust-versions.py

```diff
@@ -84,7 +84,7 @@ def test_check():
 
```

Comment:
Maybe remove the `test_check()` function. Or it will be too spammy because now you are running this for every requirements file

Reply:
ok, done